src/resources/drives.jsx

```diff
@@ -0,0 +1,83 @@
+import { 
+    List, 
+    ReferenceField,
+    TextField, 
+    NumberField,
+    Datagrid,
+    Edit,
+    SimpleForm,
+    DateField,
+    ReferenceInput,
+    TextInput,
+    BooleanField,
+    BooleanInput,
+    NumberInput,
+    Create,
+    DateTimeInput,
+    Show,
+    SimpleShowLayout
+} from "react-admin";
+
+const driveFilters = [
+  <TextInput source="q" label="Search" alwaysOn />,
+];
+
+export const DriveList = () => (
+  <List filters={driveFilters}>
+    <Datagrid rowClick="show">
+      <TextField source="id" />
+      <ReferenceField source="driver_id" reference="profiles" />
+      <DateField source="start_time" showTime />
+      <TextField source="start" />
+      <DateField source="end_time" showTime />
+      <TextField source="end" />
+      <BooleanField source="cancelled" />
+    </Datagrid>
+  </List>
+);
+
+export const DriveShow = () => (
+  <Show title={'Drive'}>
+    <SimpleShowLayout>
+      <TextField source="id" />
+      <DateField source="created_at" showTime />
+      <ReferenceField source="driver_id" reference="profiles" />
+      <DateField source="start_time" showTime />
+      <TextField source="start" />
+      <DateField source="end_time" showTime />
+      <TextField source="end" />
+      <BooleanField source="cancelled" />
+      <NumberField source="seats" min={1} max={10} />
```

Comment:
min and max here are unnecessary right?

Reply:
not sure because we had that in the app at least min is a good idea i think

Reply:
But they don't do anything for a NumberField, because you can't change it.

Reply:
yes of course sorry my mistake